client/app/components/project-geography.hbs

```diff
@@ -10,7 +10,16 @@
 {{#each @bbls as |bbl idx|}}
   <fieldset class="fieldset relative">
     <legend data-test-bbl-title="{{bbl.dcpBblnumber}}">
-      <strong>{{bbl.dcpBblnumber}}</strong>
+      <a
+        href="https://zola.planning.nyc.gov/bbl/{{bbl.dcpBblnumber}}"
+        target="_blank"
+        rel="noopener noreferrer"
```

Comment:
Out of curiosity what does this do?

Reply:
it's a linting thing -- need to add `rel="noopener noreferrer` to get linting to pass